doc/contributing/doc/README.md

```diff
@@ -33,7 +33,7 @@ Commonly one will run the following, which will go through every markdown file i
 ./util/cmdgen.py -u '**/*.md'
 ```
 
-If run, the following snippet will have the register tables of the AES block (output by `util/regtool.py -d ./hw/ip/aes/data/aes.hjson`) inserted into it.
+When run, the following snippet will have the register tables of the AES block (output by `util/regtool.py -d ./hw/ip/aes/data/aes.hjson`) inserted into it.
```

Comment:
Since we're touching this line: the text is a bit silly here. It looks like the snippet is run! Probably "When that command is run, the following snippet...." ?